examples/osm/.ipynb_checkpoints/osm_downloader-489e44fa-5ed0-4021-9184-d879e1567d35-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
Could you remove this file from this PR? If you prefer, provide a `.gitignore`.
The same for `*.csv.gz` files.